packages/core-assortments/db/helpers.js

```diff
@@ -86,6 +136,14 @@ Collections.Assortments.invalidateFilterCaches = () => {
   });
 };
 
+Collections.AssortmentLinks.findLink = ({ assortmentLinkId }) => {
+  return Collections.AssortmentLinks.findOne(assortmentLinkId);
```

Comment:
Always explicitly use the _id for security reasons, else if somehow assortmentLinkId is set to an object, this fn could leak data:

Change lines like findOne({ _id: assortmentLinkId }) to findOne(assortmentLinkId)